eng/Versions.props

```diff
@@ -8,8 +8,8 @@
     <UsingToolMicrosoftNetCompilers>true</UsingToolMicrosoftNetCompilers>
     <UsingToolSymbolUploader>true</UsingToolSymbolUploader>
     <!-- libs -->
-    <MicrosoftCodeAnalysisPooledObjectsVersion>3.3.0-beta3-19407-01</MicrosoftCodeAnalysisPooledObjectsVersion>
-    <MicrosoftCodeAnalysisDebuggingVersion>3.3.0-beta3-19407-01</MicrosoftCodeAnalysisDebuggingVersion>
+    <MicrosoftCodeAnalysisPooledObjectsVersion>3.6.0-beta1-20111-10</MicrosoftCodeAnalysisPooledObjectsVersion>
```

Comment:
@tmat Updated these to newer versions available on the current dotnet5 feeds. Original versions were from August.